src/routes/JobOpportunity/JobOpportunity.tsx

```diff
@@ -50,36 +49,157 @@ const JobOpportunity = () => {
   );
   const clusterSize = tyomahdollisuus?.jakaumat?.ammatti?.maara;
 
-  interface HeadingRoute {
-    title: string;
-    isDev?: boolean;
+  interface Section {
+    navTitle: string;
+    section: JSX.Element;
+    predicate: boolean;
```

Comment:
The predicate could perhaps be just `dev?: boolean`.

Reply:
Changed as suggested